types/reveal.js/index.d.ts

```diff
@@ -1519,10 +1534,14 @@ declare namespace Reveal {
         tokenizer?: object;
         walkTokens?: (...args: any) => any;
         xhtml?: boolean;
+        separator?: string;
```

Comment:
Can you point to where in the original library this is documented, or used in the source code?

Reply:
https://github.com/hakimel/reveal.js/blob/bbd0d3e4f73cf48a2062cacc4f7da3c25641c408/plugin/markdown/plugin.js#L101